tests/holodeck.yaml

```diff
@@ -27,6 +27,7 @@ spec:
   containerRuntime:
     install: true
     name: containerd
+    version: 1.7.27
```

Comment:
Do we want to start pointing to containerd 2.x versions? Or is the intention to test with the minimum support version of a runtime with native CDI?